packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -125,6 +125,7 @@ export const MEDIA_UPDATE_TYPE = {
   * @property {String} audio.deviceId
   * @property {Object} video
   * @property {String} video.deviceId
+  * @property {String} video.localVideoQuality // [240p, 360p, 480p, 720p, 1080p]
```

Comment:
We should remove `240p` from jsDoc

Reply:
Thanks for noticing Rajesh, I will remove that.